app/Http/Controllers/Api/TimeLog/ClockInDomain/Services/ClockInService.php

```diff
@@ -1,96 +1,32 @@
 <?php 
 namespace App\Http\Controllers\Api\TimeLog\ClockInDomain\Services;
 
-use function Opis\Closure\serialize;
-
 // Auth
 use Illuminate\Support\Facades\Auth;
-// DomainValue Objects
-use App\DomainValueObjects\UUIDGenerator\UUID;
-use App\DomainValueObjects\TimeLog\ClockIn\ClockIn;
-use App\DomainValueObjects\TimeLog\TimeStamp\TimeStamp;
-// TB Models
-use App\TimeLog;
-use App\TimeSheets;
 
-//Exception
-use App\Exceptions\TimePuncherExceptions\ClockIn\AlreadyClockedIn;
 // Contracts
 use App\Http\Controllers\Api\TimeLog\ClockInDomain\Contracts\ClockInContract;
-use App\Http\Controllers\Api\TimeLog\TimePuncher\Contracts\TimePuncherContract;
+use App\Http\Controllers\Api\TimeLog\Logic\Contracts\ClockInLogicContract;
 
 class ClockInService implements ClockInContract
```

Comment:
This service could possibly contain all the logic for clockin as opposed to 2 different services. Same goes for clockout.

Also the logic here could potentially be within the controller. Same goes for clockout